src/model.cpp

```diff
@@ -242,6 +244,55 @@ void linkComponentVariableUnits(const ComponentPtr &component)
     }
 }
 
+void findCnUnits(const XmlNodePtr &node, std::unordered_set<std::string> &cnUnits)
+{
+    XmlNodePtr childNode = node->firstChild();
+    while (childNode) {
+        if (childNode->isMathmlElement("cn")) {
+            std::string u = childNode->attribute("units");
+            if (!isStandardUnitName(u)) {
+                cnUnits.insert(u);
+            }
+        }
+        findCnUnits(childNode, cnUnits);
+        childNode = childNode->next();
+    }
+}
+
+void findComponentCnUnits(const ComponentPtr &component, std::unordered_set<std::string> &cnUnits)
+{
+    // Inspect the mathml in this component for any specified constant <cn> units,
```

Comment:
"MathML" not "mathml".

Reply:
Happy to change it if need be, but we use "mathml" throughout the rest of the code too?  This should also change?

Reply:
Change the ones in this PR. The others ought to be changed as part of issue #402 (i.e. a PR should remain on target).

Reply:
Done, ditto other places.